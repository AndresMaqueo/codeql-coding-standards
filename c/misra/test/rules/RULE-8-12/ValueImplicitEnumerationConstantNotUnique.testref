c/common/test/rules/nonuniqueenumerationconstant_shared/NonUniqueEnumerationConstant_shared.ql