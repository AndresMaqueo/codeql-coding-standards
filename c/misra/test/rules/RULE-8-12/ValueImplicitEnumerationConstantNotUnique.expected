| test.c:3:18:3:19 | c4 | Nonunique value of enum constant compared to $@ | test.c:3:22:3:23 | c5 | c5 |
